src/main/java/com/google/firebase/auth/AbstractFirebaseAuth.java

```diff
@@ -256,11 +256,13 @@ public FirebaseToken verifyIdToken(@NonNull String idToken) throws FirebaseAuthE
    * API call.
    *
    * @param idToken A Firebase ID token string to parse and verify.
-   * @param checkRevoked A boolean denoting whether to check if the tokens were revoked.
+   * @param checkRevoked A boolean denoting whether to check if the tokens were revoked or if
```

Comment:
This doesn't seem 100% clear, maybe just to me.

If it's a true/false boolean, then is it token is revoked/token is not revoked, or is it user is disabled/user is not disabled?  Can it be both somehow?

Reply:
If the developer wants to check if the token is valid, we also need to check if the user is disabled. The user can be disabled but have an unexpired ID token.

Not sure if that answers your question.

Reply:
OK got it, thanks!